### PR TITLE
Homework 5

### PR DESCRIPTION
**Pull request by michaelvolkov98**

### CODE REVIEW

File: test/specs/homework_5.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
+const { expect, assert } = require("chai");
+const homework_5_commands = require("./homework_5_commands");
+const Hw5Hotels = require("./homework_5_hotels");
+const commands = new Hw5Hotels();
```
Comment (ahsansiddiki) [inline test/specs/homework_5.js:4]: This object name `commands` is confusing as you have another class name Commnands so better to name it differently like `hotels`

File: test/specs/homework_5.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+   * Verify destination and check-in and check-out dates are as user selected
+   *
+   * 1. Launch hotels.com
+   * 2. Type "man" in destination
+   * 3. Select "Manila" from auto-suggestion
+   * 4. Select tomorrow's date as check-in date (Aug-23)
+   * 5. Select 5 days from check-in as check-out (Aug-28)
+   * 5. Click Search button
+   * 6. Verify destination has Manila
+   * 7. Verify check-in date is tomorrow's date
```
Comment (ahsansiddiki) [inline test/specs/homework_5.js:41]: Above verification is missing. You can do it by comparing with moments library.

**Review by ahsansiddiki — APPROVED** [review]

**Comment by michaelvolkov98** [comment]
"7. Verify check-in date is tomorrow's date" oops, that's not hard, i just
missed it

**Comment by michaelvolkov98** [comment]
because i used moments to get tomorrow's date lol

**Pull request merged**